config/samples/numaplane.numaproj.io_v1alpha1_isbservicerollout.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: numaplane.numaproj.io/v1alpha1
 kind: ISBServiceRollout
 metadata:
-  name: isbservicerollout-sample
-  namespace: numaplane-system
+  name: my-isbsvc
+  namespace: example-namespace
```

Comment:
seems like a more realistic example to have a separate namespace, like one would be for an Asset